renovate.json

```diff
@@ -18,5 +19,14 @@
       "matchPackageNames": ["@types/node", "node"],
       "allowedVersions": "<17.0.0"
     }
+  ],
+  "ignorePaths": [
+    "**/node_modules/**",
+    "**/bower_components/**",
+    "**/vendor/**",
+    "**/examples/**",
+    "**/__tests__/**",
+    "**/tests/**",
+    "**/__fixtures__/**"
```

Comment:
see: https://github.com/interledger/rafiki/pull/2595/files#r1532698648

basically had to carve out an exception for the test directory. renovate bot was ignoring it because of the defaults (same as here + `**/test/**`)